.github/workflows/release.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby-version: [ '2.6', '2.7', '3.0', '3.2', 'jruby' ]
```

Comment:
I would approve both changes, but we are mixing two different things on this PR:
- dependency upgrade
- new syntax